app/src/main/java/io/github/droidkaigi/confsched2018/data/repository/SponsorPlanDataRepository.kt

```diff
@@ -0,0 +1,31 @@
+package io.github.droidkaigi.confsched2018.data.repository
+
+import io.github.droidkaigi.confsched2018.data.api.DroidKaigiApi
+import io.github.droidkaigi.confsched2018.data.api.response.mapper.toSponsorPlan
+import io.github.droidkaigi.confsched2018.model.Lang
+import io.github.droidkaigi.confsched2018.model.SponsorPlan
+import io.reactivex.Completable
+import io.reactivex.Flowable
+import io.reactivex.processors.FlowableProcessor
+import io.reactivex.processors.PublishProcessor
+import javax.inject.Inject
+
+class SponsorPlanDataRepository @Inject constructor(
+        private val droidKaigiApi: DroidKaigiApi
+) : SponsorPlanRepository {
+    private val processor: FlowableProcessor<List<SponsorPlan>> = PublishProcessor.create()
+
+    override fun sponsorPlans(): Flowable<List<SponsorPlan>> = processor.toSerialized()
```

Comment:
If this repository returns api's stream, then it means app will request every time the status is changed to `active`.
To prevent that, this just returns stream.

Of course, after introducing database, this hack is not needed.